models/ajna/arbitrum/ajna_arbitrum_erc20_pools.sql

```diff
@@ -0,0 +1,43 @@
+{{  config (
+        schema = 'ajna_arbitrum',
+        alias = 'erc20_pools',
+        partition_by = ['block_time'],
+        materialized = 'incremental',
+        incremental_strategy = 'merge',
+        unique_key = ['collateral', 'quote', 'pool_address'],
```

Comment:
just double checking, are we sure all three fields are needed for unique?  or would just `pool_address` suffice?

if you change, should be universal across all chains and then the crosschain can add `blockchain` to help differentiate

Reply:
This will change to pool_address only in each chain, and `blockchain` + `pool_address` on crosschain spell